libshortfin/python/array_binding.cc

```diff
@@ -30,6 +30,14 @@ static const char DOCSTRING_ARRAY_FILL[] = R"(Fill an array with a value.
 Equivalent to `array.storage.fill(pattern)`.
 )";
 
+static const char DOCSTRING_ARRAY_VIEW[] =
+    R"(Create a view of an array.
+
+Either integer indices or slices can be passed to the view() method to create
```

Comment:
Afaict, this statement isn't correct since it is taking an explicit offset/sizes args vs a list that can either be int or slice. The latter is more compatible with prevailing Python norms, fwiw.

Reply:
int/slice makes sense. I switched back to that.